frameworks/mxnet/get_started_mnist_deploy.ipynb

```diff
@@ -0,0 +1,608 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy a Trained Model\n",
```

Comment:
```suggestion
    "# Deploy a Trained MXNet Model\n",
```

Reply:
I believe "entry point" and "endpoint" are the proper terms. I did a search in the entire `mxnet/` folder , but didn't find "entrypoint" or "end point".

Reply:
defaulted to non-local mode in both notebooks.